### ISSUE TITLE
[med] JAWS+ Chrome: Many Pages: Alt is not defined for the Avatars.

### ISSUE THREAD

**Issue by Stutikuls** [Expensify/App#8382]
## Action Performed:
1. Using JAWS+ Chrome, open URL staging.new.expensify.com 
2. Navigate to Avatar for #announce and #admins rooms using Tab key.
3. Observe the screen reader behavior on Avatar and check the code as well using developer toolbar.

## Expected Result:
Alt should be defined for the Avatar/images.

## Actual Result:
Alt is not defined for the Avatar/images. Focus does not land on the avatar/images.

HTML Code - 
div class="css-1dbjc4n r-1awozwy r-13awgt0 r-1777fci"><div class="css-1dbjc4n r-633pao">svg id="room_svg__Layer_1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 40 40" xml:space="preserve" width="80" height="80" fill="#03d47c"><style>.room_svg__st1{fill:#fff}</style><path d="M20 0c11 0 20 9 20 20s-9 20-20 20S0 31 0 20 9 0 20 0z" fill="#c6c9ca"></path><path class="room_svg__st1" d="M16.2 20.6v.6h7.5v-.6c0-1.5 1.1-2.8 2.5-3.1v-1.3c0-2.8-2.2-5-5-5h-2.5c-2.8 0-5 2.2-5 5v1.3c1.5.4 2.5 1.6 2.5 3.1z"></path><path class="room_svg__st1" d="M26.9 18.8c-1 0-1.9.8-1.9 1.9v1.9H15v-1.9c0-1-.8-1.9-1.9-1.9s-1.9.8-1.9 1.9c0 .8.5 1.5 1.2 1.8v6.3H15v-2.5h10v2.5h2.5v-6.3c.7-.3 1.2-.9 1.2-1.8.1-1.1-.8-1.9-1.8-1.9z"></path></svg></div><div dir="auto" class="css-901oao" style="color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; line-height: 20px; margin-top: 12px; max-width: 100%; text-align: center;"><span class="css-901oao css-16my406" style="color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; line-height: 20px; text-align: left;">This is the beginning of the private </span><span class="css-901oao css-16my406" style="color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; font-weight: 700; line-height: 20px; text-align: left;">#announce</span><span class="css-901oao css-16my406" style="color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; line-height: 20px; text-align: left;"> room, invite others by @mentioning them.</span>/div>/div>

**Other occurrences**
>Same issue repro on PR #9055

## Workaround:
Yes

## Area issue was found in:
Avatars on every screen.

## Failed WCAG checkpoints
1.1.1

## User impact:
Without having a text alternative, the screen reader ignores the images while reading through the page. As a result, the blind users will not get an equal opportunity to perceive the information that has been conveyed to a sighted user.

## Suggested resolution:
Provide text alternative to inline SVG by adding role="img" to the <svg>, and use the aria-label="TV Show restricted for signin" as an equivalent for alternative text.

For more information please refer to:
https://www.w3.org/WAI/WCAG21/Understanding/non-text-content

https://www.deque.com/blog/creating-accessible-svgs/

https://developer.paciellogroup.com/blog/2013/12/using-aria-enhance-svg-accessibility/

## Platform:
- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** - v1.1.46-2
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/97600950/160962592-43e04f11-12c1-43f2-8a42-b1df1c7fb208.mp4

![8187_Alt is not defined for the avatar](https://user-images.githubusercontent.com/97600950/160962603-cdeb2153-582f-4a47-8019-99c38591e154.png)
